west.yml

```diff
@@ -230,7 +230,7 @@ manifest:
       groups:
         - hal
     - name: hal_silabs
-      revision: 389726f350880238b9a1034f575ffd46c4309827
+      revision: pull/103/head
```

Comment:
Ideally, this commit should got first and also contains the required changes in CMakeLists.txt to not break the existing compilation.